src/olympia/promoted/utils.py

```diff
@@ -77,3 +77,15 @@ def retrieve_stripe_checkout_session(subscription):
     error when the session does not exist or the API call has failed."""
     stripe.api_key = settings.STRIPE_API_SECRET_KEY
     return stripe.checkout.Session.retrieve(subscription.stripe_session_id)
+
+
+def create_stripe_customer_portal(customer_id, addon):
+    """This function creates a Stripe Customer Portal object or raises an error
+    when the session does not exist or the API call has failed."""
+    stripe.api_key = settings.STRIPE_API_SECRET_KEY
```

Comment:
I notice that this isn't covered in the test. Is it worth covering, or is it just considered configuration?

Reply:
It's configuration I think